extensions/2.0/Vendor/EXT_mesh_features/README.md

```diff
@@ -126,7 +126,7 @@ The attribute's accessor `type` must be `"SCALAR"` and `normalized` must be fals
 
 *Defined in [featureIdAttribute.schema.json](./schema/featureIdAttribute.schema.json).*
 
-Per-vertex feature IDs may also be defined implicitly, as a function of vertex index within the primitive. Implicit feature IDs reduce storage costs in several common cases, such as when all vertices in a primitive share the same feature ID, or each sequential group of `N` vertices (e.g. each triangle face) share the same feature ID.
+Per-vertex feature IDs may also be defined implicitly, as a function of vertex index within the primitive. Implicit feature IDs reduce storage costs in several common cases, such as when all vertices in a primitive share the same feature ID, or each sequential group of *N* vertices (e.g. each triangle face) share the same feature ID.
```

Comment:
Elsewhere in the document we've been using code font instead of emphasis, I think we should go back to the `N` to be consistent. Make sure other places in the document are updated too. 
```suggestion
Per-vertex feature IDs may also be defined implicitly, as a function of vertex index within the primitive. Implicit feature IDs reduce storage costs in several common cases, such as when all vertices in a primitive share the same feature ID, or each sequential group of `N` vertices (e.g. each triangle face) share the same feature ID.
```